### PR TITLE
Added Node.js / MongoDB example

### PR DESCRIPTION
**Pull request by thesandlord**

### CODE REVIEW

**Comment by cjcullen** [comment]
ok to test

**Comment by cjcullen** [comment]
Travis is complaining about needing to run hack/update-generated-docs.sh. Does hack/verify-generated-docs.sh fail locally for you?

**Comment by thesandlord** [comment]
Yes it is failing.

```text
+++ [0812 10:08:29] Building go targets for darwin/amd64:
    cmd/gendocs
    cmd/genman
    cmd/genbashcomp
    cmd/mungedocs
+++ [0812 10:08:31] Placing binaries
~/kubernetes/docs/admin/namespaces/README.md
----
sync-examples:
open docs/admin/namespaces/namespace-dev.json: no such file or directory

~/kubernetes/docs/getting-started-guides/logging.md
----
sync-examples:
open examples/blog-logging/counter-pod.yaml: no such file or directory

~/kubernetes/docs/user-guide/downward-api.md
----
sync-examples:
open docs/user-guide/downward-api/dapi-pod.yaml: no such file or directory

~/kubernetes/docs/user-guide/logging.md
----
sync-examples:
open examples/blog-logging/counter-pod.yaml: no such file or directory

~/kubernetes/docs/user-guide/simple-yaml.md
----
sync-examples:
open docs/user-guide/pod.yaml: no such file or directory

~/kubernetes/docs/user-guide/walkthrough/README.md
----
sync-examples:
open docs/user-guide/walkthrough/pod-redis.yaml: no such file or directory

~/kubernetes/docs/user-guide/walkthrough/k8s201.md
----
sync-examples:
open docs/user-guide/walkthrough/pod-nginx-with-label.yaml: no such file or directory

FAIL: some manual changes are still required.
~/kubernetes/docs/ requires manual changes.  See preceding errors.
!!! Error in ./update-generated-docs.sh:28
  '"${KUBE_ROOT}/hack/after-build/update-generated-docs.sh" "$@"' exited with status 1
Call stack:
  1: ./update-generated-docs.sh:28 main(...)
Exiting with status 1
```

File: examples/nodesjs-mongodb/README.md [hunk 1]
```diff
@@ -0,0 +1,273 @@
+## Node.js and MongoDB on Kubernetes
+
+The following document describes the deployment of a basic Node.js and MongoDB web stack on Kubernetes.  Currently this example does not use replica sets for MongoDB.
+
+For more a in-depth explination of this example, please [read this post.](https://medium.com/google-cloud-platform-developer-advocates/running-a-mean-stack-on-google-cloud-platform-with-kubernetes-149ca81c2b5d)
```
Comment (corburn) [inline examples/nodesjs-mongodb/README.md]: typo: explanation

**Comment by brendandburns** [comment]
@thesandlord you need to run `/hack/update-generated-docs.sh`

**Comment by thesandlord** [comment]
@brendandburns I did, the output is attached above.

    FAIL: some manual changes are still required.
    ~/kubernetes/docs/ requires manual changes.  See preceding errors.

I didn't modify anything in the docs directory, so im not sure what I should be doing differently.

**Comment by brendandburns** [comment]
@thesandlord the error messages indicate that you have broken links.  Can you re-check the links referenced in the errors?

**Comment by thesandlord** [comment]
Fixed the generated docs issue and the spelling mistake.

**Comment by thesandlord** [comment]
Squashed commits, new pull request: https://github.com/kubernetes/kubernetes/pull/12953

**Comment by brendandburns** [comment]
@thesandlord fwiw, in the future you can squash commits and then `git push -f origin <branch>` so as to maintain the original PR...

**Comment by thesandlord** [comment]
Github automatically closed this PR when I squashed, I probably did something wrong. Will try this next time!

**Pull request closed without merging**